Makefile

```diff
@@ -18,6 +18,8 @@ bootstrap: ## Set up everything to run the app
 	poetry run pre-commit install
 	createdb notification_api || true
 	(poetry run flask db upgrade) || true
+	createdb test_notification_api || true
+	(poetry run flask db upgrade) || true
```

Comment:
We shouldn't have a command to create the test database here - `pytest` handles that for us automatically because it's also not something that should persist.  The issues folks are running into with the test DB not existing isn't due to the DB missing but another app config issue.

```suggestion
```